src/App.jsx

```diff
@@ -1,35 +1,99 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import HomePage from "./pages/HomePage";
+import Posts from "./pages/Posts";
+import Navbar from "./components/shared/Navbar";
 
-function App() {
-  const [count, setCount] = useState(0)
+import AllPostsPage from "./pages/AllPostsPage";
+import CreateEditPage from "./pages/CreateEditPostPage";
+import CreateEditUserPage from "./pages/CreateEditUserPage";
 
+import {
+  HOME_PAGE,
+  ALL_POSTS_PAGE,
+  EDIT_POST_PAGE,
+  EDIT_USER_PAGE,
+  USER_POSTS_PAGE,
+  CREATE_POST_PAGE,
+  CREATE_USER_PAGE,
+} from "./constants/routeConstants";
+
+import UserContextWrapper from "./context/UserContextWrapper";
```

Comment:
Don't use context in the file names unless the file is exporting the context.
User better naming conventions. Do some research on this.
For this we can do users -> usersListing + userForm